plugins/woocommerce/src/Admin/Features/Settings.php

```diff
@@ -61,22 +61,44 @@ public function enqueue_settings_editor_scripts() {
 		// Make sure the Settings Editor package is loaded.
 		wp_enqueue_script( 'wc-settings-editor' );
 
-		$script_handle          = 'wc-admin-edit-settings';
-		$script_path_name       = 'wp-admin-scripts/settings';
-		$script_assets_filename = WCAdminAssets::get_script_asset_filename( 'wp-admin-scripts', 'settings' );
-		$script_assets          = require WC_ADMIN_ABSPATH . WC_ADMIN_DIST_JS_FOLDER . 'wp-admin-scripts/' . $script_assets_filename;
-		$script_version         = WCAdminAssets::get_file_version( 'js', $script_assets['version'] );
-
-		wp_register_script(
-			$script_handle,
-			WCAdminAssets::get_url( $script_path_name, 'js' ),
-			$script_assets['dependencies'],
-			$script_version,
+		$script_name            = 'wc-admin-edit-settings';
+		$script_path_name       = 'settings';
+		$script_assets_filename = WCAdminAssets::get_script_asset_filename( $script_path_name, 'index' );
+		$script_assets          = require WC_ADMIN_ABSPATH . WC_ADMIN_DIST_JS_FOLDER . $script_path_name . '/' . $script_assets_filename;
+		$style_assets_filename  = WCAdminAssets::get_script_asset_filename( $script_path_name, 'style' );
+		$style_assets           = require WC_ADMIN_ABSPATH . WC_ADMIN_DIST_JS_FOLDER . $script_path_name . '/' . $style_assets_filename;
```

Comment:
I think the code could be more readable by moving these style variables closer to the style code below and maybe adding a comment to separate the two sections. What do you think?

Reply:
Good idea, created a new function in https://github.com/woocommerce/woocommerce/pull/52501/commits/323a3b9a3a68fd32a4f34aa1980f7dba96c1f88f